### ISSUE TITLE
Incorrect typescript typings for 'Mux.JWT.sign()'

### ISSUE THREAD

**Issue by MBrendel** [muxinc/mux-node-sdk#67]
Currently the typings look like this:

```typescript
export declare interface JWTOptions {
  keyId: string;
  keySecret: string;
  type: string;
  params?: any;
}

export declare class JWT {
  static sign(playbackId: string, options: JWTOptions): string;
}

```

All of the usage examples in the ReadMe (https://github.com/muxinc/mux-node-sdk#jwt-helpers-api-reference) cause typescript errors because:

- the `JWTOptions` are not optional in the signature of the `sign` method
- the `keyId ` and the `keySecret` are not optional in the `JWTOptions` type

In addition `JWTOptions` do not contain the `expiration` parameter.

Suggested types:
```typescript
export declare interface JWTOptions {
  type: string;
  keyId?: string;
  keySecret?: string;
  expiration?: string;
  params?: any;
}

export declare class JWT {
  static sign(playbackId: string, options?: JWTOptions): string;
}
```

**Comment by dylanjha** [comment]
this was closed in #69 and released in `v3.0.3` - see changelog: https://github.com/muxinc/mux-node-sdk/blob/master/CHANGELOG.md

**Comment by MBrendel** [comment]
Thanks for merging the PR!

Unfortunately I missed in my previous change, that the _type_ parameter is optional as well.

This PR  #72 fixes this and makes the 'type' type stronger too.

```typescript
export declare interface JWTOptions {
  type?: 'video' | 'thumbnail' | 'gif';
  keyId?: string;
  keySecret?: string;
  expiration?: string;
  params?: any;
}
```
